exchanges/binance/binance_wrapper.go

```diff
@@ -578,11 +578,11 @@ func (b *Binance) UpdateOrderbook(ctx context.Context, p currency.Pair, assetTyp
 		orderbookNew, err = b.GetOrderBook(ctx,
 			OrderBookDataRequestParams{
 				Symbol: p,
-				Limit:  1000})
+				Limit:  100})
 	case asset.USDTMarginedFutures:
-		orderbookNew, err = b.UFuturesOrderbook(ctx, p, 1000)
+		orderbookNew, err = b.UFuturesOrderbook(ctx, p, 100)
 	case asset.CoinMarginedFutures:
-		orderbookNew, err = b.GetFuturesOrderbook(ctx, p, 1000)
+		orderbookNew, err = b.GetFuturesOrderbook(ctx, p, 100)
```

Comment:
Can you please add some explanation on why these changes are needed?

Reply:
Spot was mostly due to the large budget increase, but there's no need for the others and the spot one is just my own personal one. Will revert 🫡